### ISSUE TITLE
Training killed without error shorting after starting

### ISSUE THREAD

**Issue by CheruscanArminius** [XPixelGroup/DiffBIR#94]
I am training the model on two nodes each with 4 A100 GPUs with the batch size of 32. Shortly after starting the training, the process gets killed. Any clue what might be the reason?

´´´bash
Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is 1024 and using 5 heads.
Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is None and using 5 heads.
Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is 1024 and using 5 heads.
DiffusionWrapper has 865.91 M params.
making attention of type 'vanilla-xformers' with 512 in_channels
building MemoryEfficientAttnBlock with 512 in_channels...
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
making attention of type 'vanilla-xformers' with 512 in_channels
building MemoryEfficientAttnBlock with 512 in_channels...
shell/5_finetune_stage2.sh: line 8:  3714 Killed                  python train.py --config configs/train_cldm.yaml
´´´

**Comment by rainsoulsrx** [comment]
> I am training the model on two nodes each with 4 A100 GPUs with the batch size of 32. Shortly after starting the training, the process gets killed. Any clue what might be the reason? I have added number of nodes by adding `num_nodes: 2` to the `train_cldm.yaml ` and requested a pull afterwards.
> 
> I executed the following command: ` python train.py --config configs/train_cldm.yaml`
> 
> ```
> {'accelerator': 'ddp', 'precision': 32, 'gpus': [0, 1, 2, 3], 'num_nodes': 2, 'default_root_dir': 'experiments/stage2', 'max_steps': 25001, 'val_check_interval': 100, 'log_every_n_steps': 50, 'accumulate_grad_batches': 4}
> Global seed set to 231
> ControlLDM: Running in eps-prediction mode
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is None and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is 1024 and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is None and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is 1024 and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is None and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is 1024 and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is None and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is 1024 and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is None and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is 1024 and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is None and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is 1024 and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is None and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is 1024 and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is None and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is 1024 and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is None and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is 1024 and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is None and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 1280, context_dim is 1024 and using 20 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is None and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is 1024 and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is None and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is 1024 and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is None and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 640, context_dim is 1024 and using 10 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is None and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is 1024 and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is None and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is 1024 and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is None and using 5 heads.
> Setting up MemoryEfficientCrossAttention. Query dim is 320, context_dim is 1024 and using 5 heads.
> DiffusionWrapper has 865.91 M params.
> making attention of type 'vanilla-xformers' with 512 in_channels
> building MemoryEfficientAttnBlock with 512 in_channels...
> Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
> making attention of type 'vanilla-xformers' with 512 in_channels
> building MemoryEfficientAttnBlock with 512 in_channels...
> Killed
> ```
> 
> Here are the output of GPU and CUDA checking right after importing pytorch.
> 
> ```
> print(torch.cuda.is_available())
> print(torch.cuda.device_count())
> print(torch.cuda.current_device())
> print(torch.cuda.get_device_name(torch.cuda.current_device())
> print(torch.cuda.memory_allocated())
> print(torch.cuda.memory_summary())
> ```
> 
> output:
> 
> ```
> 
> True
> 4
> 0
> NVIDIA A100-SXM4-80GB
> 0
> 0
> |===========================================================================|
> |                  PyTorch CUDA memory summary, device ID 0                 |
> |---------------------------------------------------------------------------|
> |            CUDA OOMs: 0            |        cudaMalloc retries: 0         |
> |===========================================================================|
> |        Metric         | Cur Usage  | Peak Usage | Tot Alloc  | Tot Freed  |
> |---------------------------------------------------------------------------|
> | Allocated memory      |       0 B  |       0 B  |       0 B  |       0 B  |
> |       from large pool |       0 B  |       0 B  |       0 B  |       0 B  |
> |       from small pool |       0 B  |       0 B  |       0 B  |       0 B  |
> |---------------------------------------------------------------------------|
> | Active memory         |       0 B  |       0 B  |       0 B  |       0 B  |
> |       from large pool |       0 B  |       0 B  |       0 B  |       0 B  |
> |       from small pool |       0 B  |       0 B  |       0 B  |       0 B  |
> |---------------------------------------------------------------------------|
> | GPU reserved memory   |       0 B  |       0 B  |       0 B  |       0 B  |
> |       from large pool |       0 B  |       0 B  |       0 B  |       0 B  |
> |       from small pool |       0 B  |       0 B  |       0 B  |       0 B  |
> |---------------------------------------------------------------------------|
> | Non-releasable memory |       0 B  |       0 B  |       0 B  |       0 B  |
> |       from large pool |       0 B  |       0 B  |       0 B  |       0 B  |
> |       from small pool |       0 B  |       0 B  |       0 B  |       0 B  |
> |---------------------------------------------------------------------------|
> | Allocations           |       0    |       0    |       0    |       0    |
> |       from large pool |       0    |       0    |       0    |       0    |
> |       from small pool |       0    |       0    |       0    |       0    |
> |---------------------------------------------------------------------------|
> | Active allocs         |       0    |       0    |       0    |       0    |
> |       from large pool |       0    |       0    |       0    |       0    |
> |       from small pool |       0    |       0    |       0    |       0    |
> |---------------------------------------------------------------------------|
> | GPU reserved segments |       0    |       0    |       0    |       0    |
> |       from large pool |       0    |       0    |       0    |       0    |
> |       from small pool |       0    |       0    |       0    |       0    |
> |---------------------------------------------------------------------------|
> | Non-releasable allocs |       0    |       0    |       0    |       0    |
> |       from large pool |       0    |       0    |       0    |       0    |
> |       from small pool |       0    |       0    |       0    |       0    |
> |---------------------------------------------------------------------------|
> | Oversize allocations  |       0    |       0    |       0    |       0    |
> |---------------------------------------------------------------------------|
> | Oversize GPU segments |       0    |       0    |       0    |       0    |
> |===========================================================================|
> ```

same question, have you solve this?
